src/Microsoft.AspNetCore.Razor.Language/Syntax/SyntaxSerializer.cs

```diff
@@ -167,6 +172,34 @@ private void WriteNode(SyntaxNode node)
                 }
             }
 
+            private void WriteRazorDirective(RazorDirectiveSyntax node)
+            {
+                if (node.DirectiveDescriptor == null)
+                {
+                    return;
+                }
+
+                var builder = new StringBuilder("Directive:{");
+                builder.Append(node.DirectiveDescriptor.Directive);
```

Comment:
Decided to write the directive descriptor info to baselines like we did before for `DirectiveChunkGenerator`. Note I'm not using the chunk generator but instead `DirectiveDescriptor` is a first class property on `RazorDirectiveSyntax`.